tests/setup/utils/await-connection.ts

```diff
@@ -7,7 +7,9 @@ export async function awaitDatabaseConnection(
 	currentAttempt = 0
 ): Promise<void | null> {
 	try {
-		await database.raw(checkSQL);
+		setTimeout(async () => {
+			await database.raw(checkSQL);
+		}, 500);
```

Comment:
Instead of delaying the individual db calls here, we should stall the whole provisioning flow right before it's calling `awaitDatabaseConnection` here:

https://github.com/directus/directus/blob/053d2a6a3761d0a1f18548349e54e41ea6799d48/tests/setup/setup.ts#L205-L218

We could change the task to something like:

```js
task: async () => {
  await sleep(25000); // 25 sec
  await awaitDatabaseConnection(knex, config.knexConfig[vendor]!.waitTestSQL)
}
```

We can use something like this as the above sleep util:

```js
function sleep(ms) {
  return new Promise(resolve => {
    setTimeout(() => {
      resolve();
    }, 5000);
  });
```